benchmarks/sharded-bm/bench.sh

```diff
@@ -617,6 +668,40 @@ monitor() {
     done
 }
 
+get_traces() {
+    fetch_forknet_details
+    echo "=> Fetching latest traces"
+    if [ -z "${TRACING_SERVER_EXTERNAL_IP}" ]; then
+        echo "Error: TRACING_SERVER_EXTERNAL_IP is not set."
+        return 1
+    fi
+
+    local CUR_TIME="$(date +%s)"
+    local LAG_SECS=${LAG_SECS:-10}
+    local LEN_SECS=${LEN_SECS:-10}
+
+    local START_TIME=$(bc <<< "$CUR_TIME - $LAG_SECS - $LEN_SECS")
+    START_TIME="$START_TIME""000"
+    local END_TIME=$(bc <<< "$CUR_TIME - $LAG_SECS")
+    END_TIME="$END_TIME""000"
+
+    echo "Current time: $CUR_TIME"
+    echo "Start time: $START_TIME"
+    echo "End time: $END_TIME"
+
+    curl -X POST http://${TRACING_SERVER_EXTERNAL_IP}:8080/raw_trace \
+        -H 'Content-Type: application/json' \
+        -d "{\"start_timestamp_unix_ms\": $START_TIME, \"end_timestamp_unix_ms\": $END_TIME, \"filter\": {\"nodes\": [],\"threads\": []}}" \
+        -o trace.json
+
+    curl -X POST http://${TRACING_SERVER_EXTERNAL_IP}:8080/profile \
+        -H 'Content-Type: application/json' \
+        -d "{\"start_timestamp_unix_ms\": $START_TIME, \"end_timestamp_unix_ms\": $END_TIME, \"filter\": {\"nodes\": [],\"threads\": []}}" \
+        -o profile.json
+
+    echo "=> Traces saved to trace.json and profile.json"
```

Comment:
nit: adding these files in `.gitignore`

nit2: add `$START_TIME` to the filename(s)?

Reply:
I'd rather give output dir as an argument instead of bloating gitignore unnecessarily. Added start time as well.